### PR TITLE
Bug score location

### PR DESCRIPTION
**Pull request by erenn16**

### CODE REVIEW

File: CaveRunner/Assets/CaveRun3D/Scripts/GameController.js [hunk 1]
```diff
@@ -99,6 +99,7 @@ function Update ()
 //This function runs at the end of the game right after you fall off the platform. It's called from inside the player script when he falls
 function EndLevel()
 {
+  showScoreOnScreen = false;
```
Comment (erenn16) [inline CaveRunner/Assets/CaveRun3D/Scripts/GameController.js]: This file has the main Score location changes.

**Comment by erenn16** [comment]
screen shot 2018-08-02 at 4 32 30 pm

**Comment by gabrielfirmacion** [comment]
I just looked at GameController.js. Looks good

**Comment by gabrielfirmacion** [comment]
👍

**Pull request merged**